lib/events/dynamic.go

```diff
@@ -104,6 +104,12 @@ func FromEventFields(fields EventFields) (AuditEvent, error) {
 			return nil, trace.Wrap(err)
 		}
 		return &e, nil
+	case UserUpdatedEvent:
+		var e events.UserCreate
```

Comment:
Actually, should this be something other than `UserCreate`?

Reply:
UserUpdate does not exist in gRPC form and is identical in format. See L92 of `lib/auth/user.go`. The UserCreate gRPC event is emitted with a modified event type.

Reply:
We merely distinguish between them for logging purposes but the data itself is identical.

Reply:
I see, let's at least add a comment about this then, otherwise it looks like a typo.